### PR TITLE
i18n: fix fr mistakes

### PR DESCRIPTION
**Pull request by MaikoTan**

### CODE REVIEW

File: src/locales/fr-FR.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 commands:
   novelai:
-    description: Générer des images sur Novel AI
+    description: Générer des images sur IA
```
Comment (MaikoTan) [inline src/locales/fr-FR.yml:3]: As we are not only supporting NovelAI now, I translate this as `Generate images from AI` to make it suitable for the usage. Comments welcome.

Comment (MaikoTan) [reply src/locales/fr-FR.yml:3]: And in french, AI is "Intelligence artificielle", so **`IA`**...

**Comment by MaikoTan** [comment]
Okay, @shigma it is good to merge.

File: src/locales/fr-FR.yml (hunk 1 above)
Comment (shigma) [reply src/locales/fr-FR.yml:3]: LGTM

**Review by shigma — APPROVED** [review]

**Pull request merged**